tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -4071,7 +4097,11 @@ TEST_P(OpConverter_FP32_FP16_INT32_Test, ConvertStridedSlice) {
           /*shrink_axis_mask=*/0,
           /*expected_output_dims=*/{1, 1, 1, 2},
           /*expected_output=*/{5, 6},
+          Status::OK(),
+          Status::OK(),
+          /*partial_input_dims=*/{1, 1, -1, -1},
       },
+      // End mask is present on just the batch dimension.
```

Comment:
Let's revise this:

An end mask is provided for the batch dimension to overwrite the end value 0 for that dimension.